scripts/qemu-check.sh

```diff
@@ -1,20 +1,46 @@
 #!/bin/bash
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright(c) 2018 Intel Corporation. All rights reserved.
+set -e
 
 SUPPORTED_PLATFORMS=(byt cht bdw hsw apl icl skl kbl cnl imx8 imx8x imx8m)
 READY_MSG="6c 00 00 00 00 00 00 70"
 
+SOF_DIR=$(cd "$(dirname "$0")" && cd .. && pwd)
+
+: "${SOF_BUILDS:=${SOF_DIR}}"
+
 rm -f dump-*.txt
 
+die()
+{
+	>&2 printf '%s ERROR: ' "$0"
+	>&2 printf "$@"
+	exit 1
+}
+
 print_usage()
 {
 	cat <<EOF
-usage: qemu-check.sh platform(s)
+usage: qemu-check.sh [ platform(s) ]
 	Supported platforms are ${SUPPORTED_PLATFORMS[*]}
+	Runs all supported platforms by default.
 EOF
 }
 
+find_qemu_xtensa()
+{
+	local xhs=xtensa-host.sh
+	for d in . ../qemu* qemu*; do
```

Comment:
query for relative path?

Reply:
I don't understand sorry.

Reply:
this part I'm not confirm, how about your current directory when run the script   
if current directory in sof, it is OK, if it is not ...

Reply:
This works in a limited number of directories, namely in `SOF/` and `SOF/*/`. Would like me to add a comment?

Reply:
I think you don't need to add this comment